etc/ATSCam/primary_hdu.header

```diff
@@ -78,6 +81,7 @@ LINSTAGE=                      / Linear Stage
 
         ---- Exposure-related information ----
 EXPTIME =                      / Exposure Time in Seconds
+SHUTTIME=                      / Shutter exposure Time in Seconds
```

Comment:
I don't think "Time" needs to be capitalized.

Reply:
fixed